### ISSUE TITLE
Validate IBAN

### ISSUE THREAD

**Issue by Herz3h** [congressus/sepa-direct-debit#26]
I tried to validate this IBAN : FR7612548029980000000150086
And it did not work, so i checked the validateiban function and it returns 65 wrong result even tho the iban is valid. So i replaced the whole 9 chunk thing by 

`$endmod = bcmod($IBANDecimal, '97');`

And it is returning 1 as expected, is there a reason you did not use this function ?

**Comment by tcheko** [comment]
Hi, 

Fails also with FR1420041010050500013M02606 which is a perfect valid IBAN number.

**Comment by JelleM-Congressus** [comment]
Hey,

Found the problem, it was an off by one error which caused the last chunk of 7 to not be processed. The bcmod approach is acceptable but I am really adamant about not using bignums/floats etc. 
The algorithm I used is straight from wikipedia and has been heavily vetted: https://en.wikipedia.org/wiki/International_Bank_Account_Number#Validating_the_IBAN however I made a stupid off by one error.

I will issue a fix later today referencing this issue.

On a side note: Security wise, posting valid IBAN numbers is not really the best move but.
